### ISSUE TITLE
Remember history of closed tabs

### ISSUE THREAD

**Issue by fgimian** [bucaran/retab#17]
Hey there, I absolutely love this plugin, it's so essential to Safari for me (coming from Chrome) :smile: 

One minor thing that I would love to see (if possible) is to have re-opened tabs restored with their history, so that the Back / Forward buttons continue to work in that tab.  I noticed that Chrome did this and sadly retab missed this feature.  I'm not sure if the JS API exposes such information, but it would make this great extension even greater.

Thanks for all your efforts!
Fotis

**Comment by bucaran** [comment]
Definitely! I tried to add this feat back since #1, but I couldn't find how to do it on that occasion so I closed it as an `unresolved crime`. I should really give it another shot. If you happen to have any idea of how this could be implemented I would love to hear ideas.

Regards

**Comment by fgimian** [comment]
@bucaran Thanks so much for your reply.  Yeah, I might give it a crack soon too, hopefully we can nail it as it would make retab that little bit more complete.  :smile:
